app/src/components/integrations/modals/addLdapIntegrationModal/twoSteps/twoStepsFooter/twoStepsFooter.jsx

```diff
@@ -0,0 +1,186 @@
+  },
+  previousStep: {
+    id: 'TwoStepsFooter.previousStep',
+    defaultMessage: 'Previous step',
+  },
+});
+
+export const createTwoStepsFooter = (
+  stepNumber,
+  onStepNumberChange,
+  onStepChange,
+  onDiscard,
+  isEdit,
+  onForceConfirmChange,
+  handleSubmit,
+  forceConfirm,
+  closeModal,
+) => {
+  return (closeHandler) => (
+    <TwoStepsFooter
+      stepNumber={stepNumber}
+      onStepNumberChange={onStepNumberChange}
+      onStepChange={onStepChange}
+      onDiscard={onDiscard}
+      isEdit={isEdit}
+      onForceConfirmChange={onForceConfirmChange}
+      handleSubmit={handleSubmit}
+      forceConfirm={forceConfirm}
+      closeModal={closeModal}
+      closeHandler={closeHandler.closeHandler}
+    />
+  );
+};
+
+export const TwoStepsFooter = ({
+  stepNumber,
+  onStepNumberChange,
+  onStepChange,
+  onDiscard,
+  isEdit,
+  onForceConfirmChange,
+  handleSubmit,
+  forceConfirm,
+  closeModal,
+  closeHandler,
+}) => {
+  const { formatMessage } = useIntl();
+
+  useEffect(() => {
+    closeModal && closeHandler();
+  }, [closeModal]);
+
+  useEffect(() => {
+    !closeModal && forceConfirm && handleSubmit();
+  }, [forceConfirm]);
+
+  const onNext = () => {
+    onStepChange(stepNumber + 1);
+  };
+
+  const isFirstStep = stepNumber === 1;
+
+  const getSubmitButtonText = () => {
+    if (isFirstStep) {
+      return formatMessage(messages.nextStep);
+    } else if (isEdit) {
+      return formatMessage(messages.saveAndClose);
+    } else {
+      return formatMessage(messages.create);
+    }
+  };
+
+  const saveAndClose = () => {
+    onForceConfirmChange(true);
+  };
+
+  return (
+    <div className={cx('two-steps-footer')}>
+      {isEdit ? (
+        <>
+          <Button
+            className={cx('discard-button')}
+            variant={'ghost'}
+            onClick={onDiscard}
+            data-automation-id="discardButton"
+          >
+            {formatMessage(messages.discardChanges)}
+          </Button>
+          {isFirstStep && (
+            <Button
+              variant={'ghost'}
+              onClick={saveAndClose}
+              data-automation-id="saveAndCloseButton"
+            >
+              {formatMessage(messages.saveAndClose)}
+            </Button>
+          )}
+        </>
+      ) : (
+        <Button
+          className={cx('cancel-button')}
+          variant={'ghost'}
+          onClick={closeHandler}
+          data-automation-id="cancelButton"
+        >
+          {formatMessage(messages.cancel)}
+        </Button>
+      )}
+      {stepNumber === 2 && (
+        <Button variant={'ghost'} onClick={() => onStepNumberChange(1)}>
+          <i className={cx('left-icon')}>{Parser(RightArrowIcon)}</i>
+          {formatMessage(messages.previousStep)}
+        </Button>
+      )}
+      <Button onClick={onNext} data-automation-id="submitButton">
+        {getSubmitButtonText()}
+        {isFirstStep && <i className={cx('right-icon')}>{Parser(RightArrowIcon)}</i>}
```

Comment:
Can we use svg icons from ui-kit?

Reply:
There is no such arrow there.

Reply:
It can be simply added 
https://github.com/reportportal/ui-kit/pull/33/commits/e16383345eac5ff5dd6e6d7f6e168b30f1f5447f